community-code/How_to_Monitor_Diagnose_and_Solve_Gradient_Issues_in_Foundation_Models/requirements.txt

```diff
@@ -0,0 +1,18 @@
+# Core ML libraries
+torch==1.10.2+cu115  # CUDA 11.5-compatible version
```

Comment:
Since users might have a different version of CUDA installed, this should ideally be parameterized in the notebook, or they should be informed about installing a version compatible with their CUDA version